### ISSUE TITLE
TypeError: undefined is not a constructor while using ValueListFieldType on ESNext

### ISSUE THREAD

**Issue by talmosko** [remult/remult#203]
**Describe the bug**
when setting the target in tsconfig.json to be ESNext:
`  "target": "ESNext",
` the use of "ValueListFieldType" decorator throws this error:

```
API] [INFO] 12:40:38 Restarting: /home/talm/Documents/Coding/Remult/react-vite-express-starter/src/server/api.ts has been modified
[API] TypeError: undefined is not a constructor
[API]     at Function.<static_initializer> (/home/talm/Documents/Coding/Remult/react-vite-express-starter/src/shared/Stage.ts:17:17)
[API]     at Object.<anonymous> (/home/talm/Documents/Coding/Remult/react-vite-express-starter/src/shared/Stage.ts:12:19)
[API]     at Module._compile (node:internal/modules/cjs/loader:1254:14)
[API]     at Module._compile (/home/talm/Documents/Coding/Remult/react-vite-express-starter/node_modules/source-map-support/source-map-support.js:568:25)
[API]     at Module.m._compile (/tmp/ts-node-dev-hook-18396826836177915.js:69:33)
[API]     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
[API]     at require.extensions..jsx.require.extensions..js (/tmp/ts-node-dev-hook-18396826836177915.js:114:20)
[API]     at require.extensions.<computed> (/tmp/ts-node-dev-hook-18396826836177915.js:71:20)
[API]     at Object.nodeDevHook [as .ts] (/home/talm/Documents/Coding/Remult/react-vite-express-starter/node_modules/ts-node-dev/lib/hook.js:63:13)
[API]     at Module.load (node:internal/modules/cjs/loader:1117:32)
[API] [ERROR] 12:40:38 TypeError: undefined is not a constructor
```

works on target = "es5"

**To Reproduce**
create an entity with ValueListFieldType, for example:

```
import { ValueListFieldType } from "remult";

@ValueListFieldType({
  getValues: () => [
    Stage.seed,
    Stage.seriesA,
    Stage.seriesB,
    Stage.seriesC,
    Stage.IPO,
  ],
})
export class Stage {
  constructor(public id: number, public caption: string) {
    this.id = id;
    this.caption = caption;
  }
  static seed = new Stage(1, "Seed");
  static seriesA = new Stage(2, "Series A");
  static seriesB = new Stage(3, "Series B");
  static seriesC = new Stage(4, "Series C");
  static IPO = new Stage(5, "IPO");
}
```
register it:

```
export const api = remultExpress({
  entities: [Company, Stage],
});
```
set the target to `ESNext`

and get the error

**Expected behavior**
The error shouldn't be thrown while using ESNext as a target

**Comment by noam-honig** [comment]
I tried to reproduce the error and didn't get it - can you share the repo where this error occurred?

Also, note that in the code you've shared - you do not need to register stage as an Entity - it's just a value type.

**Comment by talmosko** [comment]
@noam-honig 
https://github.com/talmosko/remult-react-table-example
pushed right now.

btw - node v18.16.0

**Comment by noam-honig** [comment]
Please make me a contributor:
noam-honig

**Comment by noam-honig** [comment]
This was a typescript issues with version 4.9, it was fixed in version 5.1 and up
